spinalcordtoolbox/registration/landmarks.py

```diff
@@ -288,14 +286,14 @@ def getRigidTransformFromLandmarks(points_dest, points_src, constraints='Tx_Ty_T
         ax.set_aspect('auto')
         plt.legend()
         # plt.show()
-        plt.savefig(os.path.join(path_qc, 'getRigidTransformFromLandmarks_plot.png'))
+        plt.savefig('getRigidTransformFromLandmarks_plot.png')
 
         fig2 = plt.figure()
         plt.plot(sse_results)
         plt.grid()
         plt.title('#Iterations: ' + str(res.nit) + ', #FuncEval: ' + str(res.nfev) + ', Error: ' + str(res.fun))
         plt.show()
-        plt.savefig(os.path.join(path_qc, 'getRigidTransformFromLandmarks_iterations.png'))
+        plt.savefig(os.path.join('getRigidTransformFromLandmarks_iterations.png'))
```

Comment:
Nit: There is an extra `os.path.join` here.

Reply:
sorry no time to wait for new ci--